04_CollectionTypes.playground/Contents.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+/*
+ MARK: - 🏠 Домашнее задание.
+ 
+ TODO: - 🧑‍💻 Выполните каждый из пунктов для закрепления материала. Решение пишите на следующей строке после задания:
+ // - проинициализируйте два пустых массива foodArray and drinkArray разными способами
+ */
+//var foodArray: [String] = []
+//var drinkArray: [String] = []
+//взял в кавычки так как выдавало ошибку
+
+ // - присвойте первому массиву стринг значения - названия продуктов (минимум 5)
+var foodArray: [String] = ["burger", "sandwich", "frenchfries", "chocolate", "cheese", "sausage"]
+
+ // - присвойте второму массиву названия напитков (минимум 5)
+var drinkArray: [String] = ["milk", "cola" , "coffee", "beer" , "water", "juice"]
+
+ // - добавьте в конец foodArray еще одно значение
+foodArray.insert("butter", at: 6)
```

Comment:
What if `foodArray` has more then 5 or even less then 5 items? 
When you insert at: 6 where 6 is an index u must be careful, if `foodArray` has 4 items and u will try to insert at 6 app will crash. 
check pls how `.append()` works. 
or if u would like to use `.insert` u have to use not 6 but `drinkArray.count` to guarantee that u won't have out if range crash.
`drinkArray.insert("butter", at: drinkArray.count)`